lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -271,14 +271,14 @@ function _campaign_resource_retrieve($nid) {
 /**
  * Retrieve and show index list response of campaigns requested.
  *
- * @param string $ids
- * @param bool $staff_pick
- * @param bool $mobile_app
- * @param string $mobile_app_date
- * @param string $term_ids
- * @param int $count
- * @param bool $random
- * @param int $page
+ * @param  string  $ids
```

Comment:
Just my two cents, I tend to not do it in our Java code because you'll have wildly different name sizes for your types sometimes which I think can be weird. So you could see something like...

```
* @param bool                is_diego
* @param CampaignTransformer transformer
* @param int                 some_num
* @param Campaign            campaign
```

... which, eh I guess isn't terrible.